modules/client/examples/01-client/13-prepare-installation.ts

```diff
@@ -5,18 +5,18 @@ title: PrepareInstallation
 ### Prepare installation of a plugin
 */
 
+import { Client } from "@aragon/sdk-client";
 import {
-  Client,
   PrepareInstallationParams,
   PrepareInstallationStep,
-} from "@aragon/sdk-client";
+} from "@aragon/sdk-client-common";
 import { context } from "../index";
 
 // Instantiate the general purpose client from the Aragon OSx SDK context.
 const client: Client = new Client(context);
 
 // Obtained from the plugin metadata
-const installationAbi = [ ... ];
+const installationAbi = [/* ... */];
```

Comment:
Ideally also a real example of an installation ABI so devs know what to expect for format

Reply:
Even if it's just a link to an example somewhere or a link to documentation on how to retrieve it  - that would suffice. The important thing we want to avoid is a developer reaching this point, and not understanding what it is, what format it should be in or where they should get this information. We should always assume devs are juniors to ensure getting this up and running is as easy as possible for everyone.

Reply:
There's now a link to https://github.com/aragon/simple-storage-example-plugin/blob/main/contracts/release1/build3/build-metadata-R1B3.json#L7-L20

I personally recommend avoiding to add visual noise like this
<img width="588" alt="image" src="https://github.com/aragon/sdk/assets/4456749/d80c1d45-e24b-4a88-878b-f884c4738314">